docker-compose.yml

```diff
@@ -6,9 +6,7 @@ services:
     working_dir: /kvrocks
     volumes:
         - ./storage:/kvrocks/conf
-    ports:
-        - 6101:6101
```

Comment:
Instead of removing it, can you replace it by `6101:6666` so we don't need a dedicated config file and keep using the default one? (my goal is to keep exposing the same port as before, I think it is the way to do that)

Reply:
It is not necessary to expose the port because they are already on the same network. If you want to change the port, it will be more at the level of the kvrocks configuration. But, there is an issue with their Dockerfile as they are hardcoding the port : https://hub.docker.com/layers/apache/kvrocks/latest/images/sha256-c73f96ce8429613b5e2af2691afa3bd3f92c3b1c488c7b2c30ef9e0b2a412128?context=explore